diagnostic.md

```diff
@@ -3,36 +3,40 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+1. Organizing code into modules.
+2. Setting up build tools.
+3. Client side rendering.
```

Comment:
Other advantages include:
Front end routing
Easily display dynamic content using two-way binding
No more AJAX - Ember takes care of making API calls for you.

1/1